### PR TITLE
Bug_fixes_align_header_of_context_menu

### PR DESCRIPTION
**Pull request by datnguyenhuy**
image
image
image
image

### CODE REVIEW

File: lib/presentation/model/file/document_presentation_file.dart [hunk 1]
```diff
@@ -102,6 +102,7 @@ class DocumentPresentationFile extends Equatable implements PresentationFile {
 
   @override
   Widget fileIcon() {
+    print(mediaType.getFileTypeImagePath(imagePath));
```
Comment (hoangdat) [inline lib/presentation/model/file/document_presentation_file.dart:105]: please remove that

File: lib/presentation/model/file/presentation_file.dart [hunk 2]
```diff
@@ -42,4 +42,6 @@ abstract class PresentationFile {
   int fileSize();
 
   Widget fileIcon();
+
+  String fileIconImagePath();
```
Comment (hoangdat) [inline lib/presentation/model/file/presentation_file.dart:46]: why we need this method?

File: lib/presentation/view/header/context_menu_header_builder.dart [hunk 3]
```diff
@@ -40,18 +40,21 @@ class ContextMenuHeaderBuilder {
...
+        child: ListTile(
+          key: _key,
+          leading: Transform.translate(
+              offset: _file.fileIconImagePath() == 'assets/images/ic_folder.svg' ? Offset(0,0) : Offset(0,2),
+              child: _file.fileIcon()
           ),
-        ),
-      trailing: _file.fileSize() == 0 ? null : Text(filesize(_file.fileSize()), style: TextStyle(fontSize: 14, color: AppColor.documentModifiedDateItemTextColor)));
+          title: Text(
+            _file.fileName(),
```
Comment (hoangdat) [inline lib/presentation/view/header/context_menu_header_builder.dart:53]: How about if `fileName` is too long?

File: lib/presentation/view/header/context_menu_header_builder.dart [hunk 4]
```diff
@@ -40,18 +40,21 @@ class ContextMenuHeaderBuilder {
...
-          _file.fileName(),
-          maxLines: 1,
-          style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+  Transform build() {
+    return Transform.translate(
+        offset: Offset(0,3.5),
+        child: ListTile(
+          key: _key,
+          leading: Transform.translate(
+              offset: _file.fileIconImagePath() == 'assets/images/ic_folder.svg' ? Offset(0,0) : Offset(0,2),
```
Comment (imGok) [inline lib/presentation/view/header/context_menu_header_builder.dart:49]: Use constants instead please and I think you `fileIconImagePath` is not needed

Comment (imGok) [reply lib/presentation/view/header/context_menu_header_builder.dart:49]: You could do : `_file is WorkGroupFolderPresentationFile ? ..` or smth like this wdyt ?

Comment (datnguyenhuy) [reply lib/presentation/view/header/context_menu_header_builder.dart]: It was so good that I couldn't think of it. Thank you very much

File: lib/presentation/model/file/presentation_file.dart (hunk 2 above)
Comment (datnguyenhuy) [reply lib/presentation/model/file/presentation_file.dart]: It is not needed and i removed it. Thank you, please

File: lib/presentation/view/header/context_menu_header_builder.dart (hunk 3 above)
Comment (datnguyenhuy) [reply lib/presentation/view/header/context_menu_header_builder.dart:55]: I added the ellipsis at the end if it's too long

File: lib/presentation/view/header/context_menu_header_builder.dart [hunk 5]
```diff
@@ -40,18 +42,22 @@ class ContextMenuHeaderBuilder {
...
-    return ListTile(
-      key: _key,
-      leading: _file.fileIcon(),
-      title: Transform(
-        transform: Matrix4.translationValues(-16, -2, 0.0),
-        child: Text(
-          _file.fileName(),
-          maxLines: 1,
-          style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+  Transform build() {
```
Comment (hoangdat) [inline lib/presentation/view/header/context_menu_header_builder.dart:45]: Can you explain the different from List tile with Transform

Comment (datnguyenhuy) [reply lib/presentation/view/header/context_menu_header_builder.dart:45]: ListTile is used to display the list box
Transform is used to align ListTile on the header context menu, taking ListTile as a child widget

**Review by hoangdat — APPROVED** [review]

**Pull request merged**